racket/collects/racket/promise.rkt

```diff
@@ -212,3 +212,27 @@
                     (cons '#:tick       #'0.2)
                     (cons '#:use        #'0.12))])
     (λ(stx) (make-delayer stx #'delay/idle kwds))))
+
+;; A lighter-weight version of delay that doesn't care about multiple values or
+;; exceptions
+
+;; force/delay* : Delay*-Promise -> Any
+;; only deals with one value and doesn't care about exceptions
+(define (force/delay* promise)
+  (let ([v (pref promise)])
+    (cond [(pair? v)
+           (unsafe-car v)]
```

Comment:
Fixed now. I had used unsafe because the [similar code](https://github.com/AlexKnauth/racket/blob/e08188aeda8c419b741313ef34ef0e49ac6eb6a4/racket/collects/racket/private/promise.rkt#L83) in `private/promise.rkt` used unsafe. Should I change that too?